src/Utility/Xml.php

```diff
@@ -277,6 +277,7 @@ public static function fromArray($input, array $options = [])
         ];
         $options += $defaults;
 
+        /** @var DOMDocument|\DOMElement $dom */
```

Comment:
Backslash missing before `DOMDocument`. It's strange the var needs to be hinted this way when it's obviously a `DOMDocument` instance.

Reply:
The whole DOM and in general PHP Stuff is really badly written, lots of legacy I am sure.

Reply:
Yeah the DOM classes are really wacky.